modules/calypso-redirects/calypso-redirects.css

```diff
@@ -0,0 +1,43 @@
+/* All Menu Indicators */
+a[href$="calypso-plugins"] .wp-menu-name::after,
```

Comment:
These links are using underscores, so the styles are not being applied correctly for me 

![screen shot 2017-10-17 at 12 39 37 pm](https://user-images.githubusercontent.com/7129409/31677288-49ecac66-b338-11e7-8e1f-81a57aa188b5.png)